omnibus-optimism-sepolia.toml

```diff
@@ -0,0 +1,112 @@
+name = "synthetix-omnibus"
+version = "4"
+description = "Includes the full synthetix system with configurations applied"
+include = [
+    "tomls/settings.toml",
+    "tomls/ccip-testnet.toml",
+    "tomls/core.toml",
+    "tomls/permissions.toml",
+    "tomls/collaterals/snx.toml",
+    "tomls/pools/spartan-council.toml",
+    "tomls/permit-all-transferCrossChain.toml",
+    "tomls/markets/spot-factory.toml",
+    "tomls/markets/spot/eth.toml",
+]
+
+[setting.snx_package]
+defaultValue = "synthetix:3.3.5"
+
+[setting.spot_market_package]
+defaultValue = "synthetix-spot-market:3.3.5"
+
+[setting.owner]
+defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
+
+[setting.salt]
+defaultValue = "0x3330363734343032330000000000000000000000000000000000000000000000"
```

Comment:
I reckon this was the salt we used on mainnet. May be better to change to smth else?